### ISSUE TITLE
Want to help out

### ISSUE THREAD

**Issue by VenkatSubramaniam** [JuliaImages/Tinker.jl#14]
Hey! 
I was an intern with Julia Computing in Bangalore over the summer, and I've recently got pretty interested in image processing in general. I've got some extra time on my hands and was just scouring the web for something new to work on when I saw this.  I was wondering--is there any way that I could help you out with this project?

**Comment by timholy** [comment]
Hi @VenkatSubramaniam, I was deferring to others here but it looks like the main author of Tinker isn't a watcher for the package (probably because I was the one who created it). Let's ping @kswanson33 to rope everyone into the conversation.

This is just to say that I'd be excited to collaborate on this, and I think Tinker could definitely help bring JuliaImages and Julia itself to a wider audience. @kswanson33 can probably comment better on the current state of Tinker, but while there's obviously a lot still to do there's clearly some good stuff brewing.

**Comment by kswanson33** [comment]
Thanks for the ping, @timholy! @VenkatSubramaniam I'm thrilled you're interested! I'm a full-time student and since the semester started again I haven't had the same kind of time to work on this project. I was midway through catching up with the documentation when things got busy; I'll push what I have so far to a branch, to be finished & merged when I have the time. Hopefully that will be helpful. (Warning: I'm a fairly inexperienced coder, so if it looks like I've made a silly programming choice odds are I have. :) )
If you have any questions let me know. I might also open a few issues when I get around to it.

**Comment by hearnsj** [comment]
Bumping this issue up. It would be nice to get Tinker working with Julia 1.x

**Comment by timholy** [comment]
Agreed, interest in this topic seems to be heating up; I've had about four conversations in the last two weeks regarding interest in a ImageJ-like GUI for JuliaImages. @hearnsj, care to give a go at updating this package via Julia 0.7?

**Comment by hearnsj** [comment]
Well I have made a start by forking the repository!
